ghcjs/github.json

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
Is this gone because we're using upstream's ghcjs?

Reply:
Yes - we are just using what's in nixpkgs.

Reply:
Great